master (#5395)
treeverse/lakeFS: docs/reference/authorization.md

Each policy attached to a user or a group has an `Effect` - either `allow` or `deny`.
During evaluation of a request, `deny` would take precedence over any other `allow` policy.

> If no permissions exist than the action will be blocked, making no permission the equivilent of `deny`. 

Reviewer: This is rather confusing to me, although I understand why it is correct. Deny is an explicit operation, whereas no permissions is a default. They are not equivalent: for instance, adding a single Allow permission yields a different result from each. They do have equal effects.
Perhaps 
```suggestion
The default if no permissions exist is to allow nothing.
```

This helps us compose policies together. For example: 

* we could attach a very permissive policy to a user and use `deny` rules to then selectively restrict what that user can do.

> * Since `deny` is stronger than `allow`, you could specify `fs:*` `allow` with `fs:Write*` `deny` to a user with the net effect that the user will be able to do all fs commands _except_ write commands.

Reviewer: I would strongly like to discourage doing this.  For instance, `fs:Create*` and `fs:Delete*` will still be allowed using this configuration, including `fs:CreateMetaRange`.  So if you gave me this policy, I could probably do almost everything you think you are preventing.

I would prefer to give an example of denying by path.  So you could allow me to do some things on a repo, but prevent me from doing anything under some prefix.
